frontend/src/ui/Blocks/Series.tsx

```diff
@@ -242,30 +577,33 @@ const UpcomingEventsGrid: React.FC<React.PropsWithChildren> = ({ children }) =>
             backgroundColor: "var(--grey86)",
             borderRadius: 4,
             marginBottom: 8,
```

Comment:
There needs to be more margin above. I like `margin-top: 8`.

![image](https://user-images.githubusercontent.com/7419664/218829887-1fd5c174-8280-426b-964d-6bbc4517564d.png)